charmcraft.yaml

```diff
@@ -96,6 +96,10 @@ config:
       type: secret
       description: >
         The Juju secret ID corresponding to the password used by the agents to connect to Wazuh.
+    api-password:
+      type: secret
+      description: >
+        The Juju secret ID corresponding to the password used by the wazuh API.
```

Comment:
Maybe we should update the description here to mention that this will only apply to the first OpenSearch database connected. If the user removes the OpenSearch integration and re-integrates with another OpenSearch database, the password won’t change, as this only occurs during the `install` event.

One solution I can think of is to turn this into an action that the user can run to change the password to a randomly generated one and receive the new password as output.

Reply:
The action would only work once, otherwise I'd have to store the password somewhere else since it is needed for authenticating the request. That's increasing significantly the attack surface. I'm expanding the config description